src/main/java/in/org/projecteka/hiu/SecurityConfiguration.java

```diff
@@ -85,6 +86,7 @@
             INTERNAL_PATH_PATIENT_CARE_CONTEXT_INFO,
             PATH_ON_AUTH_INIT,
             PATH_ON_AUTH_CONFIRM,
+            GET_CONSENT_CERT,
```

Comment:
Similarly this one can be renamed to `GET_CERT`.

Reply:
done